VALITATION-GUIDE.md

```diff
@@ -0,0 +1,320 @@
+# Guia de validação
```

Comment:
Shouldn't this file be named `VALIDATION-GUIDE.md`? I guess `VALIDATION` might be a typo.